landoapi/hg.py

```diff
@@ -91,6 +99,21 @@ class PatchConflict(PatchApplicationFailure):
     )
 
 
+def check_fix_output_for_replacements(fix_output: List[bytes]) -> Optional[List[str]]:
+    """Parses `hg fix` and returns a list of changeset hashes, or None
+    if no changesets are changed.
```

Comment:
The summary line should be a single line (see pep 257)

Reply:
Should flake8 and/or black have caught this?

Reply:
Looks like there's at least a [discussion](https://github.com/psf/black/issues/144) about it. Not sure if anything was decided.

Reply:
nit:
```suggestion
    """Parse `hg fix` output.
    
    Returns:
        A list of changeset hashes, or None if no changesets are changed.
```